cmd/drone-docker/main.go

```diff
@@ -102,6 +102,16 @@ func main() {
 			Usage:  "don't start the docker daemon",
 			EnvVar: "PLUGIN_DAEMON_OFF",
 		},
+		cli.BoolFlag{
```

Comment:
Maybe use `BoolTFlag` so that it defaults to `true` and drop the `DISABLE` naming?

Reply:
I implemented this per the above linked Gitter convo with @bradrydzewski, so I'll defer to him on that. Happy to make the change if he's cool with it.